Source/santad/EventProviders/SNTEndpointSecurityFileAccessAuthorizer.mm

```diff
@@ -310,6 +310,48 @@ - (FileAccessPolicyDecision)specialCaseForPolicy:(std::shared_ptr<WatchItemPolic
   return FileAccessPolicyDecision::kNoPolicy;
 }
 
+/// An An `es_process_t` must match all criteria within the given
+/// WatchItemPolicy::Process to be considered a match.
+- (bool)policyProcess:(const WatchItemPolicy::Process &)policyProc
+     matchesESProcess:(const es_process_t *)esProc {
+  if (esProc->codesigning_flags & CS_SIGNED) {
+    // Check if the instigating process has an allowed TeamID
+    if (policyProc.team_id.length() > 0 && esProc->team_id.data &&
+        policyProc.team_id != esProc->team_id.data) {
+      return false;
+    }
```

Comment:
nit: team ID is checked last elsewhere in the code. 

In this case it doesn't matter as we said that a process has to match *all* specified fields.

Reply:
Spoke offline. Will leave as-is since it's in order of expected evaluation cost.